test/unit/core.ts

```diff
@@ -848,7 +847,7 @@ describe("snabbdom", function () {
               return spanNumWithOpacity(n, "1");
             }),
           );
-          const shufArr = shuffle(arr.slice(0));
+          const shufArr = arr.slice(0).sort(() => 0.5 - Math.random());
```

Comment:
This "[Microsoft shuffle](https://www.robweir.com/blog/2010/02/microsoft-random-browser-ballot.html)" is not a good way of shuffling an array. See also [the comments on this SO answer](https://stackoverflow.com/a/18650169/556072).

If we want to remove the Lodash dependency I think we should define our own `shuffle` function that does a proper shuffle. The implementation in the bottom of [this SO answer](https://stackoverflow.com/a/12646864/556072) seems like a fine candidate.

Reply:
the SO shuffle is added here, but maybe lodash-shuffle was better than using a custom function.

feel free to close this PR or merge either way